<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="baseLeft">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>53</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Заделка слева</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="baseRight">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Заделка справа</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>74</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>37</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">L</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>sigma</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>q</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F</string>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="showElement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>92</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрисовка стержней</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Calc</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>556</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="Nx">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N(X)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Ux">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>U(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Gx">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gx</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>5</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>L</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N(X)</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="UXresultWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>L</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>U(x)</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="GxresultWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>L</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>G(x)</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главное</string>
    </property>
    <addaction name="open"/>
    <addaction name="write"/>
    <addaction name="action_3"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Стержни</string>
    </property>
    <addaction name="add"/>
    <addaction name="remove"/>
    <addaction name="removeAll"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="open">
   <property name="text">
    <string>Открыть прокт</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Закрыть проект</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="add">
   <property name="text">
    <string>Добавить стержень</string>
   </property>
  </action>
  <action name="remove">
   <property name="text">
    <string>Удалить стержень</string>
   </property>
  </action>
  <action name="removeAll">
   <property name="text">
    <string>Удалить все стержни</string>
   </property>
  </action>
  <action name="calc">
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </action>
  <action name="write">
   <property name="text">
    <string>Сохранить проект</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
